docs/user/security/session-management.asciidoc

```diff
@@ -0,0 +1,60 @@
+[role="xpack"]
+[[xpack-security-session-management]]
+=== Session management
+
+When you log in, {kib} creates a session that is used to authenticate subsequent requests to {kib}. A session consists of two components: an encrypted cookie that is stored in your browser, and an encrypted document in a dedicated {es} hidden index. By default, the name of that index is `.kibana_security_session_1`, where the prefix is derived from the primary `.kibana` index. If either of these components are missing, the session is no longer valid.
+
+When your session expires, or you log out, {kib} will invalidate your cookie and remove session information from the index. {kib} also periodically invalidates and removes any expired sessions that weren't explicitly invalidated.
+
+[[session-idle-timeout]]
+==== Session idle timeout
+
+You can expire sessions after a period of inactivity. This and `xpack.security.session.lifespan` are both
+highly recommended. By default, session doesn't expire because of inactivity. To define a sliding session expiration, set
+the `xpack.security.session.idleTimeout` property in the `kibana.yml`
+configuration file. The idle timeout is formatted as a duration of
+`<count>[ms|s|m|h|d|w|M|Y]` (e.g. '70ms', '5s', '3d', '1Y'). For example, set
+the idle timeout to expire sessions after 1 hour of inactivity:
```

Comment:
It feels odd to see `lifespan` before `idleTimeout`, even though this is the idle timeout section.

How about rewording it like this?

```suggestion
You can use `xpack.security.session.idleTimeout` to expire sessions after a period of inactivity. This and `xpack.security.session.lifespan`
are both highly recommended.

By default, sessions don't expire because of inactivity. To define a sliding session expiration, set the  property in the `kibana.yml`
configuration file. The idle timeout is formatted as a duration of `<count>[ms|s|m|h|d|w|M|Y]` (e.g. '20m', '24h', '7d', '1w'). For example,
set the idle timeout to expire sessions after 1 hour of inactivity:
```
Note 1: changed wording "session doesn't expire" -> "sessions don't expire"
Note 2: changed duration example to match that in prior comments